src/Microsoft.TestPlatform.CommunicationUtilities/Resources/Resources.Designer.cs

```diff
@@ -79,6 +78,15 @@ public class Resources {
             }
         }
         
+        /// <summary>
+        ///   Looks up a localized string similar to Cannot attach the debugger to the default test host..
+        /// </summary>
+        public static string AttachDebuggerToDefaultTestHostFailure {
```

Comment:
>AttachDebuggerToDefaultTestHostFailure [](start = 29, length = 38)

hmm, looks very similar to the string in CrossPlat engine. Was this intentional cause we don't have the PID here?

Reply:
Yes, the PID is an implementation detail of `AttachDebuggerToTestHost` and it's completely transparent to the caller.

Reply:
So we can't reuse the string cause we don't have a PID where this string is used? Not super particular.. Just wanted to see if there was a way to reuse.